docs/guidelines.md

```diff
@@ -113,7 +113,30 @@ ngeo.exampleDirective = function(…) {
       // …
     },
     controllerAs: 'ctrl',
+    // …
+  });
+```
+
+But if you bind your directive to the controller and you declare your attribute
+as a two ways binding attrribute, you can use the `.` notation. See the example
```

Comment:
More or less *by chance*. My guess is that the properties in `scope` are not renamed because there are other properties with the same name that are exported or in an externs file. @gberaudo, what do you think?